### DISCUSSION TITLE
Mutual Close issue with funds stuck as "waiting close" in Limbo

### DISCUSSION THREAD

**Discussion by jtymoszczuk** [Ride-The-Lightning/RTL#839] [Q&A] [answered]
My funds are stuck in a channel with Wallet os Satoshi.  It says its in "waiting close". This is listed under pending but is not pending when i check BOS command line

I tried to Coop close 2 channels i opened with wallet of satoshi on around Thursday the 14th. so it's been a few days. In Ride the Lightning UI Its listed under Peers/Channels -> Pending -> Waiting Close. The funds have not cleared. Im not sure how to get funds to clear.

I opened a channel twice with WOS.  After learning it would be more beneficial to have one large channel i decided to close both of them as a mutual close in Thunderhub and reopen one large one. But the 2 channels never closed so i didn't want to open anything until i receive those funds back.

I couldn't find the funds using command line but it shows in RTL UI. I use BOS sometimes but mostly RTL and Thunderhub.
i attached the pictures of the transaction from Ride The Lightning. Do you know what command line would find this in Balance of Satoshi? Do you know how i can clear the funds or force close channel if it isn't closed?

Transaction ID
4e1fa3a52e5996d516cd30f918904ffad24bb44b54f51d5d2f9374309961f5b4
Channel Point
912628f78600cc820b7a47a19f115f97f715e380e07298af8cdff8503715df06:1

![RTL](https://user-images.githubusercontent.com/83677877/137772139-05dd9279-d8d5-4fc9-a25b-9bb91d6426dd.png)

![RTL tx details 1](https://user-images.githubusercontent.com/83677877/137772602-51367f28-e170-4965-9fe8-7664067ad589.png)

![RTL tx details 2](https://user-images.githubusercontent.com/83677877/137772634-b36fd24c-3115-40bd-95d6-c447f6d51a88.png)

**Comment by saubyk** [comment]
Converting this issue into a discussion.

**Comment by saubyk** [comment]
@jtymoszczuk it appears that the channel closing transactions have not been initiated from LND. I couldn't find your transactions in mempool.
As a first course of action, I would recommend restarting LND.

**Reply by jtymoszczuk** [reply]
I use umbrel and have restarted it a couple times. same thing. nothing changes

**Reply by maximlomans** [reply]
try
$ ls -l ~/.loop/mainnet 
see lsat.token.pending ? 
 mv it to lsat.token.pending.bkp extension but to another folder not .loop/mainnet, then reboot 
and read the docs all of them bitcoin's and lightning network

"https://wiki.ion.radar.tech/ ", 
"https://lightningwiki.net/index.php/Main_Page" , 
"https://lightning.engineering/loopapi/#loop-grpc-api-reference" ,
 "https://github.com/lightninglabs/loop/blob/master/docs/autoloop.md ",
 "https://en.bitcoin.it/wiki/Main_Page ".

good luck 
; )

**Reply by saubyk** [reply]
> try $ ls -l ~/.loop/mainnet see lsat.token.pending ? mv it to lsat.token.pending.bkp extension but to another folder not .loop/mainnet, then reboot and read the docs all of them bitcoin's and lightning network
> 
> "https://wiki.ion.radar.tech/ ", "https://lightningwiki.net/index.php/Main_Page" , "https://lightning.engineering/loopapi/#loop-grpc-api-reference" , "https://github.com/lightninglabs/loop/blob/master/docs/autoloop.md ", "https://en.bitcoin.it/wiki/Main_Page ".
> 
> good luck ; )

Unsure, if this response applies to the discussion thread here. were you trying to post this on the loop query?

**Comment by jtymoszczuk** [comment, marked as answer]
just ended up force closing by 
~/umbrel/bin/lncli closechannel --force (channel point id with a space instead of “:”

Example:
`~/umbrel/bin/lncli closechannel --force 898b88e2abc2a1d7d61f55fc826522cdddf9779f8b547cd2f84ff37253413e2f 1`

Find Channel point by `~/umbrel/bin/lncli  pendingchannels`

Hope that helps anyone that comes across this issue. more of an LND issue than a RTL
